generators/server/templates/src/main/java/package/Application.java.ejs

```diff
@@ -117,7 +123,8 @@ public class <%= mainClass %> {
                 "Application '{}' is running! Access URLs:\n\t" +
                 "Local: \t\t{}://localhost:{}{}\n\t" +
                 "External: \t{}://{}:{}{}\n\t" +
-                "Profile(s): \t{}\n----------------------------------------------------------",
+                "Profile(s): \t{}\n\t" +
+                "VM timezone: \t{}\n----------------------------------------------------------",
```

Comment:
It might be better to just say timezone since you won't always be in a VM when running a JHipster app.

Reply:
fixed